model_sync_transactions_test.go

```diff
@@ -101,3 +101,27 @@ func Test_areParentsBroadcast(t *testing.T) {
 		})
 	}
 }
+
+func TestSyncTransaction_SaveHook(t *testing.T) {
+	t.Parallel()
+
+	t.Run("trim Results to last 20 messages", func(t *testing.T) {
+		ctx, client, deferMe := CreateTestSQLiteClient(t, false, true, WithCustomTaskManager(&taskManagerMockBase{}))
```

Comment:
Our Code Standards say that we should have
// Given
...
// When
...
// Then
...
in our tests :)